bip-0009.mediawiki

```diff
@@ -120,7 +124,7 @@ adding three to the current calendar year when drafting the soft fork proposal):
 
 If the soft fork still not ''locked-in'' and the
 GetMedianTimePast() of a block following a retarget period is at or
-past this timeout, miners should cease setting this bit.
+past this timeout, miners SHOULD cease setting this bit.
```

Comment:
I'm not so sure this is a MUST. If it is a MUST I think we need to specify for how long they need to cease setting the bit since eventually the bit might get reused at which point the correct behavior for a client unaware of the new rule change is to accept the block but warn the user.